media/css/base/page-not-found.scss

```diff
@@ -2,79 +2,70 @@
 // License, v. 2.0. If a copy of the MPL was not distributed with this
 // file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-@import '../pebbles/includes/lib';
-@import '../hubs/sections';
+@import '../../protocol/css/includes/lib';
 
-.mzp-c-navigation + #outer-wrapper,
-.moz-global-nav-drawer + #outer-wrapper {
-    padding-top: 40px;
+$font-path: '/media/fonts';
+$image-path: '/media/protocol/img';
+
+.list-content a:link, .list-content a:visited, .list-content a:hover, .list-content a:active {
+    color: black;
 }
 
 main {
-    min-height: 450px;
-
-    header {
-        margin-bottom: 40px;
+    font-size: 18px;
+}
 
-        h1 {
-            @include font-size-huge;
-            color: #fff;
-            line-height: 1.3;
+.error_header {
+    font-size: 40px;
+}
 
-            span {
-                @include box-decoration-break(clone);
-                background-color: #000;
-            }
-        }
+.list-icon {
+    vertical-align: top;
+    padding-right: $spacing-md;
+}
 
-        p {
-            max-width: 35em;
-        }
-    }
+.has-icon {
+    padding-bottom: $spacing-md;
+}
 
-    .error-content {
-        max-width: 35em;
-    }
+.back-button:before {
+    content:"\2190\0020";
+}
 
-    .fx-privacy-link a {
-        @include font-size-extra-small;
+.back-button {
+    text-decoration: none;
+    color: $color-link;
+    cursor: pointer;
+}
 
-        &:link,
-        &:visited {
-            color: $color-link;
-        }
+.hide-back {
+    display: none;
```

Comment:
Instead of hiding this link conditionally if there's no history, might it be worth hiding it by default (since it relies on JS to work), and then only showing it if there's something to go back to?